src/components/scenes/GuiPluginListScene.js

```diff
@@ -235,35 +234,37 @@ class GuiPluginList extends React.Component<Props, State> {
     this.showCountrySelectionModal().catch(showError)
   }
 
-  _renderPlugin = ({ item }) => {
+  renderPlugin = ({ item }) => {
     const { pluginId } = item
     const plugin = guiPlugins[pluginId]
+    const styles = getStyles(this.props.theme)
 
     return (
-      <TouchableWithoutFeedback onPress={() => this.openPlugin(item).catch(showError)}>
-        <View style={styles.pluginRow}>
+      <View style={styles.pluginRowContainer}>
+        <TouchableOpacity onPress={() => this.openPlugin(item).catch(showError)}>
           <View style={styles.pluginRowLogoAndInfo}>
-            <View style={styles.logo}>
-              <Image style={styles.logoImage} source={paymentTypeLogosById[item.paymentTypeLogoKey]} />
-            </View>
-            <View style={styles.textBoxWrap}>
-              <Text style={styles.titleText}>{item.title}</Text>
-              <Text style={styles.subtitleText}>{item.cryptoCodes.length > 0 && `Currencies: ${item.cryptoCodes.join(', ')}`}</Text>
-              <Text style={styles.subtitleText}>{item.description}</Text>
+            <Image style={styles.logo} source={paymentTypeLogosById[item.paymentTypeLogoKey]} />
+            <View style={styles.pluginTextContainer}>
+              <EdgeText style={styles.titleText}>{item.title}</EdgeText>
+              <EdgeText style={styles.subtitleText} numberOfLines={0}>
+                {item.cryptoCodes.length > 0 && `Currencies: ${item.cryptoCodes.join(', ')}`}
+              </EdgeText>
+              <EdgeText style={styles.subtitleText}>{item.description}</EdgeText>
             </View>
           </View>
           <View style={styles.pluginRowPoweredByRow}>
-            <Text style={styles.footerText}>Powered by </Text>
+            <EdgeText style={styles.footerText}>{s.strings.plugin_powered_by + ' '}</EdgeText>
             <Image style={styles.partnerIconImage} source={{ uri: item.partnerIconPath }} />
```

Comment:
Partner icons also need to be a part of the theme so we can have a light mode versions